techniques/html/H2.html

```diff
@@ -114,25 +114,31 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+      <h2>解説</h2>
+      <p>このテクニックの目的は、テキストとアイコンの両方でリンクを提供する際、キーボード利用者や支援技術の利用者にとって混乱や困難を招くような ウェブページを作らないようにすることである。様々な利用者がテキストとアイコンを使いやすいと見出していることから、両方を提供することでリンクのアクセシビリティを向上させることができる。</p>
+      <p>よくあるリンクとして、テキストとアイコンの両方が互いに隣接しながらも、別々の <code class="language-html">a</code> 要素としてレンダリングされているものがある。視覚的にはそれらは単一のリンクのように見えるが、それらを同様のリンクが隣接しているものとして受け取る利用者も多い。キーボード利用者の場合、冗長なリンクを通過していくのは面倒である。支援技術の利用者にとって、連続する同様のリンクに遭遇すると混乱する可能性がある。アイコンのテキストによる代替がリンクテキストの複製である場合、スクリーンリーダーは読み上げを 2 度繰り返すことになる。</p> 
+      <p>コンテンツ制作者がリンク画像の代替テキストを省略した場合、テキストによる代替が視覚的なリンクと同じ目的を果たさないため、<a href="https://waic.jp/translations/WCAG22/Understanding/non-text-content.html">達成基準 1.1.1</a> を満たせなくなる。</p> 
+      <p>このテクニックは、テキストと画像を一つの <code class="language-html">a</code> 要素にまとめ、画像に空の代替テキストを指定してテキストの重複を排除することによって、混乱や困難がないリンクを提供するものである。このようにすることで、リンクにおける両方の表現が提供されつつも、キーボード利用者にとってリンクは一つだけとなり、ウェブページのリンク先リストを利用者に提供する補助技術も重複リンクを含まなくなる。</p>
+      <p>ページをレイアウトしやすくするために、テキストとアイコンのリンクを隣接したテーブルセルに分けることがある。WCAG 2 ではレイアウトテーブルの使用を禁止していないが、HTML の table 要素に定義されたセマンティックな意味を保持させ、コンテンツから提示を分離するコーティング手法に準拠するためにも、<abbr title="Cascading Style Sheets">CSS</abbr> ベースのレイアウトが推奨されている。CSS が使用されている場合には、このテクニックを適用して、テキストとアイコンのリンクを一つにまとめることができる。</p>
+      <div class="note">
+        <p class="note-title marker">訳注</p>
+        <div>
+          <p>WAIC では H2 に関する<a href="https://waic.jp/guideline/as/">アクセシビリティ サポーテッド（AS）情報</a>を提供している。2020 年 3 月版の<a href="https://waic.jp/docs/as/info/techs/H2.html">アクセシビリティ サポーテッド（AS）情報: H2</a> も参照。
+        </div>
+      </div>
    </section>
    <section id="examples">
-      <h2>Examples</h2>
-      <section class="example" id="example-1"><h3>Example 1</h3>
+      <h2>事例</h2>
+      <section class="example" id="example-1"><h3>事例 1</h3>
          
-         <p>The icon and text are contained in the same <code class="language-html">a</code> element.</p>
+         <p>アイコンとテキストが同じ <code class="language-html">a</code> 要素の中にある。</p>
          
 <pre xml:space="preserve"><code class="language-html">&lt;a href="products.html"&gt;
   &lt;img src="icon.gif" alt=""&gt;Products page
 &lt;/a&gt;</code></pre>
       </section>
-      <section class="example" id="example-2"><h3>Example 2</h3>
-         <p>A link contains an icon and text, and the site help refers to the icon. The <code class="language-html">img</code> has a text alternative which is the name used for the icon in the site help, which describes clicking the home page icon.</p>
+      <section class="example" id="example-2"><h3>事例 2</h3>
+         <p>リンクにアイコンとテキストがあり、サイトのヘルプでそのアイコンの説明をしている。<code class="language-html">img</code> には、サイトのヘルプで使われている名前であるテキストによる代替があり、ホームページのアイコンをクリックすることが説明されている。/p>
```

Comment:
`</p>` の閉じタグの一部が欠けていました。
```suggestion
         <p>リンクにアイコンとテキストがあり、サイトのヘルプでそのアイコンの説明をしている。<code class="language-html">img</code> には、サイトのヘルプで使われている名前であるテキストによる代替があり、ホームページのアイコンをクリックすることが説明されている。</p>
```